solution/backend/regcore/search/migrations/0008_alter_searchquery.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.18 on 2023-04-12 15:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
The search vector field is a generated column.  In order to alter a column related to it you need to drop the column and then repopulate it.  Legnth had to be modified for section string so the column is just dropped, field altered, then readded with existing calculations.